server/http_handler_test.go

```diff
@@ -1199,3 +1199,48 @@ func TestSetLabels(t *testing.T) {
 	// reset the global variable
 	config.GetGlobalConfig().Labels = map[string]string{}
 }
+
+func TestSetLabelsConcurrentWithGetLabel(t *testing.T) {
```

Comment:
This test can also passed before this PR.

Reply:
Updated.